### DISCUSSION TITLE
this may be a FF limitation but would love a fix if someone has it

### DISCUSSION THREAD

**Discussion by scottedwards2000** [aaFn/Bookmark-search-plus-2#308] [Q&A]
I love being able to open the UI as a tab and pin it to the browser so I can always hit command-1 and go back to my bookmarks, but for some reason every time I restart FF, the tab doesn't reopen.  Right now, I just quickly hit command-shift-T to bring it back, but does anyone know is this some limitation with tabs whose URLS start with moz-extension:// ?

**Comment by aaFn** [comment]
Hello @scottedwards2000 , that works for me, at least on Windows:
- Open BSP2 as a tab
- Pin the tab
- Go to some other tab
- Close FF
- Open FF again
- The pinned tab with BSP2 inside is still there and easy to switch to

**Reply by scottedwards2000** [reply]
thanks for the reply – unfortunately this does NOT work on mac it appears... oh well, luckily if I hit command-shift-T after startup, the BSP2 tab reopens and is still pinned - so not too bad :-)

**Comment by aaFn** [comment]
Ok, maybe file a bug on bugzilla to FF, or verify if the bug is not already open ?
https://bugzilla.mozilla.org/home
